actors/builtin/miner/miner_actor.go

```diff
@@ -1417,146 +1708,47 @@ func validateExpiration(rt Runtime, st *State, activation, expiration abi.ChainE
 		rt.Abortf(exitcode.ErrIllegalArgument, "invalid expiration %d, total sector lifetime (%d) cannot exceed %d after activation %d",
 			expiration, expiration-activation, sealProof.SectorMaximumLifetime(), activation)
 	}
-
-	// ensure expiration is one epoch before a proving period boundary
-	periodOffset := st.ProvingPeriodStart % WPoStProvingPeriod
-	expiryOffset := (expiration + 1) % WPoStProvingPeriod
-	if expiryOffset != periodOffset {
-		rt.Abortf(exitcode.ErrIllegalArgument, "invalid expiration %d, must be immediately before proving period boundary %d mod %d",
-			expiration, periodOffset, WPoStProvingPeriod)
-	}
```

Comment:
I think we should actually add this back (clamping to epoch before deadline boundary)

Reply:
Does that really help? We can't clamp it to the _right_ deadline.